ui/models/submissionAction.md

```diff
@@ -0,0 +1,23 @@
+# Submission Action
+
+This describes an action that the user must perform on this submission before it can be accepted. The honest broker or metadata curator will do their work and create these objects where necessary to inform the user that they must do something.
```

Comment:
```suggestion
This describes an action that the user must perform on this submission before it can be accepted, or identifies changes made by the honest broker or curator. The honest broker or metadata curator will do their work and create these objects where necessary to inform the user that they must do something.
```